app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -118,27 +154,61 @@ function useGrantDetail() {
   });
   const grantMetadata = computed(() => (grant.value ? metadata.value[grant.value.metaPtr] : null));
 
+  // --- Check properties ---
+  const hasWebsite = computed(() =>
+    grant.value
+      ? metadata.value[grant.value.metaPtr]?.properties?.projectWebsite !== undefined &&
+        metadata.value[grant.value.metaPtr]?.properties?.projectWebsite !== ''
+      : false
+  );
+  const hasGithub = computed(() =>
+    grant.value
+      ? metadata.value[grant.value.metaPtr]?.properties?.projectGithub !== undefined &&
+        metadata.value[grant.value.metaPtr]?.properties?.projectGithub !== ''
+      : false
+  );
+  const hasHandle = computed(() =>
+    grant.value
+      ? metadata.value[grant.value.metaPtr]?.properties?.twitterHandle !== undefined &&
+        metadata.value[grant.value.metaPtr]?.properties?.twitterHandle !== ''
+      : false
+  );
+
   // --- Edit capabilities ---
   const isOwner = computed(() => userAddress.value === grant.value?.owner);
   const isEditing = ref(false);
 
-  const form = ref<{ owner: string; payee: string; name: string; description: string }>({
+  const form = ref<{
+    owner: string;
+    payee: string;
+    name: string;
+    description: string;
+    website: string;
+    github: string;
+    handle: string;
```

Comment:
Nit: Maybe `twitter` or `twitterHandle` instead to be more explicit? I can imagine adding e.g. a telegram handle field down the road